merge.py

```diff
@@ -6,7 +6,7 @@
 out = open(sys.argv[3], "w")
 
 for l in inf:
-    if (l.find("rfcdiff.js") != -1):
+    if (l.find('<script src="rfcdiff.js"></script>') != -1):
```

Comment:
My argument against this change is that it will not properly rewrite a script line that is an absolute URL to a site. Is your plan to instead have some sort of conditional statement in the script?